### ISSUE TITLE
Do not print format warning when using format parameter

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/mixer-tools#532]
```
+ mixer build update --offline --config builder.conf --min-version 27210 --format 27 --no-publish --no-signing --native
Warning: Using FORMAT value from /usr/share/defaults/swupd/format
```
When passing the `--format 27` option at the command line, it does not appear to be honored and instead, mixer appears to use the system-installed format.  The intended behavior is that anything defined on the command line overrides anything in a config file or any fallback pattern.

**Comment by rchiossi** [comment]
From what I can see in the code, mixer is using the value in `--format`, the problem is that the warning is misplaced. This warning is printed when mixer creates a new mixer.state file, but that happens before the format value is overwritten by the command line parameter. Behavior is good, we need to fix the error message.

**Comment by reaganlo** [comment]
Fixed in v6.2.4
